### ISSUE TITLE
Move to ASP.NET Core

### ISSUE THREAD

**Issue by aitchDashGee** [nkdAgility/Practical-DevOps-Workshop#4]
Would't it make sense to have all exercises use ASP.NET Core?

**Comment by MrHinsh** [comment]
I think that it would. My only concern is that ASP.NET Core is currently a moving target...

**Comment by aitchDashGee** [comment]
I can certainly see that one, but I think by doing this and integrating Docker into the actual devops pipeline inside VSTS, you will get a more complete "VSTS as the ultimate cross-platform orchestration tool" story.

**Comment by MrHinsh** [comment]
Agreed... I am attempting to see what a refactor looks like if I add pluggable samples. That way Teams or Instructors can choose which sample to follow...

**Comment by aitchDashGee** [comment]
That would be pretty neat, yup! :)
